lib/swaps/errors.ts

```diff
@@ -4,6 +4,7 @@ const codesPrefix = errorCodesPrefix.SWAPS;
 const errorCodes = {
   SWAP_CLIENT_NOT_FOUND: codesPrefix.concat('.1'),
   SWAP_CLIENT_NOT_CONFIGURED: codesPrefix.concat('.2'),
+  SWAP_CLIENT_RAIDEN_DISABLED: codesPrefix.concat('.3'),
```

Comment:
Per @erkarl's comments, it sounds like we'd just want to use the `SWAP_CLIENT_NOT_FOUND` error since clients that are disabled won't be tracked in the swap client manager. More generally speaking though, I think the errors in the swaps module should be generic and not specify particular client types like Raiden.